Improved Great People 8

etiennefd opened the pull request:
The main change is that I split Holy Rome and Germany, and added many names to both. The split is from 18th century onwards, so all pre-1700 Germans are in the Holy Rome list, as are post-1700 people who are more closely associated with Austria (including a few Hungarians and Czechs). Post-1700 people who are more closely associated with Prussia or modern Germany are in the Germany list. 

Things are not always clear cut for 18th c. people who were not Prussian or Austrian (Schiller? Goethe?). One possibility would be to make the split only from the industrial era onwards. Let me know. I don't claim any sort of particular knowledge of German culture and history, so I might have made some questionable choices. One thing I'm sure of is that Gutenberg and Kafka really needed to be added to the list. 

Other changes:
- Ethiopia: removed Ezana and Menelik, added Haile Selassie as a prophet.
- Added Publicola's suggestions for Mongolian GPs.
- Moved the Maya and China to preserve spawn order.
- Various other additions and corrections.

etiennefd committed:
Split Germans and Austrians, added others

etiennefd committed:
More German/Austrian GP

etiennefd committed:
Lucy Maud Montgomery as Canadian GA

etiennefd committed:
Mongol GPs

etiennefd committed:
Ottoman GPs

Removed the title "Pasha" from all names

etiennefd committed:
Russia

etiennefd committed:
Merge remote-tracking branch 'upstream/develop' into improved_gp_8

etiennefd committed:
Ethiopian GP

Removed Menelik and Ezana, added Haile Selassie as a prophet

etiennefd committed:
Moved Germany

etiennefd committed:
Moved Maya list

etiennefd committed:
Medieval scientists

etiennefd committed:
Portuguese, English

etiennefd committed:
Mughal poets

etiennefd committed:
Phoenician names spelling

etiennefd committed:
Merge remote-tracking branch 'upstream/develop' into improved_gp_8

etiennefd committed:
Moved China in the list

dguenms commented:
Haven't looked into the changes in depth, but a possible solution for the neither Prussian nor Austrian GPs is to just include them in both lists. The game will still consider an already used name taken and not spawn them for the other civ.

etiennefd commented:
That's probably a good idea. I'll go over the lists and change them accordingly. 

> Le 3 févr. 2019 à 11:57, Daniel Günther <notifications@github.com> a écrit :
> 
> Haven't looked into the changes in depth, but a possible solution for the neither Prussian nor Austrian GPs is to just include them in both lists. The game will still consider an already used name taken and not spawn them for the other civ.
> 
> —
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub <https://github.com/dguenms/Dawn-of-Civilization/pull/264#issuecomment-460068368>, or mute the thread <https://github.com/notifications/unsubscribe-auth/AGGQG7gwj8m65ganwI4P_mtylzPrhyK4ks5vJxTggaJpZM4agLLv>.
>

etiennefd committed:
Duplicated a few German names

i.e. 18th century Germans who are clearly aligned with neither Prussia nor Austria

etiennefd committed:
added two 18th c. Austrians

dguenms commented on Assets/Python/GreatPeople.py:
```diff
@@ -3263,13 +3312,19 @@ def setup():
 		iIndustrial,
 		"Balingiin Tserendorj", # 19th
+		iGlobal, 
+		"Damdin Sükhbaatar", # 20th
```
u""

dguenms commented on Assets/Python/GreatPeople.py:
```diff
@@ -2493,68 +2515,76 @@ def setup():
 	iGreatScientist : [
-		"Nikolaus von Kues", # 15th
+		"Hrabanus Maurus", # 9th
+		"fHildegard von Bingen", # 12th
```
Hildegard von Bingen should stay a prophet imo

etiennefd committed:
Requested changes
